funnel/views/email.py

```diff
@@ -139,6 +150,51 @@ def send_email_for_organization_admin_membership_revoked(
     )
 
 
+@signals.user_registered_for_project.connect
+def send_email_for_project_registration(project, user):
+    background_project_registration_email.queue(
+        project_id=project.id, user_id=user.id, locale=get_locale()
+    )
+
+
+@signals.user_cancelled_project_registration.connect
+def send_email_for_project_deregistration(project, user):
```

Comment:
The first parameter to both signals should be the `rsvp` object, not the project. Send project as a separate parameter.